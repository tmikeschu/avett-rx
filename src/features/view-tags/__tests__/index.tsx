import * as React from "react";

import * as utils from "test";

import ViewTags from "../";

describe("<ViewTags />", () => {
  it("renders the tags", async () => {
    const { findByText } = utils.render(<ViewTags />);

    expect(await findByText("ðŸ˜­")).toBeInTheDocument();
  });
});
