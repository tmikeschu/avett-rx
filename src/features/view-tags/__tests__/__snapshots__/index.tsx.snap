// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewTags /> shows a loading spinner while fetching data 1`] = `
<div>
  <div
    class="chakra-spinner css-1qrt4ic"
  >
    <span
      class="css-f8n5zr"
    >
      Loading...
    </span>
  </div>
</div>
`;

exports[`<ViewTags /> shows an empty state 1`] = `
<div>
  <p
    class="chakra-text css-1aut27h"
  >
    Oh snap! We don't have any tags to show yet.
  </p>
</div>
`;

exports[`<ViewTags /> shows an error state 1`] = `
<div>
  <p
    class="chakra-text css-1asabzw"
  >
    Oh no! Something went wrong fetching that data.
  </p>
</div>
`;

exports[`<ViewTags /> shows data after loading 1`] = `
<ul
  aria-label="tags list"
  class="css-1ttwu97"
  role="list"
>
  <li
    class="css-1jwz7be"
  >
    ðŸ¥°
  </li>
  <li
    class="css-1jwz7be"
  >
    ðŸ˜­
  </li>
  <li
    class="css-1jwz7be"
  >
    ðŸ˜Š
  </li>
  <li
    class="css-1jwz7be"
  >
    ðŸ˜”
  </li>
  <li
    class="css-1jwz7be"
  >
    ðŸ¤ª
  </li>
</ul>
`;
